docs/setup/install/targz.asciidoc

```diff
@@ -26,12 +26,13 @@ The Linux archive for Kibana v{version} can be downloaded and installed as follo
 ["source","sh",subs="attributes"]
 --------------------------------------------
 wget https://artifacts.elastic.co/downloads/kibana/kibana-{version}-linux-x86_64.tar.gz
-sha1sum kibana-{version}-linux-x86_64.tar.gz <1>
+wget https://artifacts.elastic.co/downloads/kibana/kibana-{version}-linux-x86_64.tar.gz.sha512
+shasum -a 512 -c kibana-{version}-linux-x86_64.tar.gz.sha512 <1>
 tar -xzf kibana-{version}-linux-x86_64.tar.gz
 cd kibana-{version}-linux-x86_64/ <2>
 --------------------------------------------
-<1> Compare the SHA produced by `sha1sum` or `shasum` with the
-    https://artifacts.elastic.co/downloads/kibana/kibana-{version}-linux-x86_64.tar.gz.sha1[published SHA].
+<1> Compares the SHA of the downloaded `.tar.gz` archive and the published checksum, which should output
+    `kibana-{version}-linux-x86_64.tar.gz: OK`.
```

Comment:
The {version} variable doesn't seem to work with inline code blocks.  @debadair any ideas if there's a workaround?  I couldn't find anything.